src/box/lua/config/applier/box_cfg.lua

```diff
@@ -341,9 +347,29 @@ local function apply(config)
         box_cfg.listen = box.NULL
     end
 
-    -- Construct box_cfg.replication.
-    if box_cfg.replication == nil then
+    -- luacheck: ignore 542 empty if branch
+    if configdata:get('replication.disabled', {use_default = true}) then
```

Comment:
Let's save the configuration value into a variable to use in the second patch (where we forbid to start without a local snapshot).

Reply:
Done.